packages/jest-jasmine2/src/jasmine/Env.ts

```diff
@@ -78,15 +87,21 @@ export default function (j$: Jasmine) {
     clearReporters: () => void;
     addReporter: (reporterToAdd: Reporter) => void;
     it: (description: string, fn: QueueableFn['fn'], timeout?: number) => Spec;
-    xdescribe: (description: string, specDefinitions: Function) => Suite;
+    xdescribe: (
+      description: string,
+      specDefinitions: SpecDefinitionsFn,
+    ) => Suite;
     xit: (description: string, fn: QueueableFn['fn'], timeout?: number) => Spec;
     beforeAll: (beforeAllFunction: QueueableFn['fn'], timeout?: number) => void;
     todo: () => Spec;
     provideFallbackReporter: (reporterToAdd: Reporter) => void;
     allowRespy: (allow: boolean) => void;
-    describe: (description: string, specDefinitions: Function) => Suite;
+    describe: (
+      description: string,
+      specDefinitions: SpecDefinitionsFn,
+    ) => Suite;
 
-    constructor(_options?: object) {
+    constructor(_options?: Record<string, unknown>) {
```

Comment:
This isn't actually used anywhere, and removing them doesn't make TS sad nor require changes in any other packages.
I'm assuming it's there to mirror `jasmine`, but happy to remove if folks want :)

(this involves [these](https://github.com/facebook/jest/pull/10216/files#diff-0fb06d0f4dae8d82ca5a856def1039cfR48) two [lines](https://github.com/facebook/jest/pull/10216/files#diff-8af29d62178b94f59e9adc208a67755bR82))

Reply:
We should remove in Jest 27 - could you open a PR removing it? I'll add it to the milestone so we don't forget

Reply:
Done: #10240